lib/item.php

```diff
@@ -357,5 +357,11 @@ public function save()
             $sql->addGlobalUpdateFields($user);
             $sql->insert();
         }
+
+        rex_extension::registerPoint(new rex_extension_point(
+            'FEEDS_ITEM_SAVED',
+            [$this->streamId, $this->uid]
```

Comment:
Ich weiß jetzt nicht so genau, für welche Zwecke die EPs so gedacht sind, aber finde es etwas merkwürdig, die Daten so als `subject` zu übergeben. Ist das Absicht?
Ansonsten würde ich sie eher als Params mit benannten Keys übergeben (als dritter Parameter).

Reply:
ich weiß nicht, was best practice ist. Ein Blick in die EPs vom Core in der Doku legen nahe, dass es nicht immer ein subject gibt - soll dann einfach dazwischen `null` stehen?

Der EP ist da, um im Anschluss etwas anzustoßen, bezogen auf den soeben aktualisierten Datensatz.

Reply:
Siehe gepushte Änderungen.

Ich habe gesehen, es gibt eine Property `  -readonly: false`, die sollte bei beiden EP-Typen true sein, oder?

Reply:
> Siehe gepushte Änderungen.

Sieht gut aus!

> Ich habe gesehen, es gibt eine Property ` -readonly: false`, die sollte bei beiden EP-Typen true sein, oder?

Denke ja, wobei es hier nicht wirklich wichtig ist.
Ich glaube `readonly` sorgt vor allem dafür, dass die Extensions das `subject` nicht ändern können. Da du das aber ja gar nicht hast (`null`), ist es hier also unwichtig.